cluster/gce/util.sh

```diff
@@ -1205,7 +1205,17 @@ function test-teardown {
     --project "${PROJECT}" \
     --quiet \
     "${NODE_TAG}-${INSTANCE_PREFIX}-nodeports" || true
-  "${KUBE_ROOT}/cluster/kube-down.sh"
+  if [[ ${MULTIZONE:-} == "true" ]]; then
+      local zones=( ${E2E_ZONES} )
+      # tear them down in reverse order, finally tearing down the master too.
+      for ((zone_num=${#zones[@]}-1; zone_num>0; zone_num--))
+      do
+	  KUBE_GCE_ZONE="${zones[zone_num]}" KUBE_USE_EXISTING_MASTER="true" "${KUBE_ROOT}/cluster/kube-down.sh"
+      done
+      KUBE_GCE_ZONE="${KUBE_GCE_ZONE:-us-central1-b}" "${KUBE_ROOT}/cluster/kube-down.sh"
```

Comment:
actually, shouldn't this just be `KUBE_GCE_ZONE="${zones[0]}"`? trying to read `$KUBE_GCE_ZONE` is just going to get you the last value set in the loop, I think, which will be `${zones[1]}`. (maybe also explicitly set `KUBE_USE_EXISTING_MASTER="false"` just to be safe, in case it's set in the environment already?)

(yet again another comment on this line. I'm sorry. too many ongoing reviews for me.)